tests/integration/tier0/non-destructive/config-file/test_config_file.py

```diff
@@ -2,64 +2,107 @@
 
 from collections import namedtuple
 
+import pytest
+
 
 Config = namedtuple("Config", "path content")
 
 
-def create_files(config):
+def _create_files(config):
     for cfg in config:
         with open(os.path.expanduser(cfg.path), "w") as f:
             f.write(cfg.content)
 
         os.chmod(os.path.expanduser(cfg.path), 0o600)
 
 
-def remove_files(config):
+def _remove_files(config):
     for cfg in config:
         os.remove(os.path.expanduser(cfg.path))
 
 
-def test_config_custom_path_custom_filename(convert2rhel):
-    config = [Config("~/.convert2rhel_custom.ini", "[subscription_manager]\nactivation_key = config_activationkey")]
-    create_files(config)
-
+@pytest.fixture
+def set_up_config_files(request):
+    config_files = request.param
+    _create_files(config_files)
+
+    yield
+
+    _remove_files(config_files)
+
+
+@pytest.mark.parametrize(
+    "set_up_config_files",
+    [
+        [
+            Config(
+                "~/.convert2rhel_custom.ini",
```

Comment:
It is a small thing, but I think we can try to have a custom config file name that douesn't contain the convert2rhel in its name. Let's try something like

```suggestion
                "~/.custom_foo_config.ini",
```